networking/net.go

```diff
@@ -49,23 +41,37 @@ func LoadNets() ([]Net, error) {
 		return nil, err
 	}
 
-	var nets []Net
-
+	files := []string{}
 	for _, dent := range dirents {
 		if dent.IsDir() {
 			continue
 		}
 
-		nf := path.Join(RktNetPath, dent.Name())
+		files = append(files, dent.Name())
+	}
+
+	return files, nil
+}
+
+func LoadNets() ([]Net, error) {
+	files, err := listFiles(RktNetPath)
+	if err != nil {
+		return nil, err
+	}
+
+	sort.Strings(files)
+
+	var nets []Net
```

Comment:
Why do you do files := []string{} but var nets []Net? :stuck_out_tongue: 
Might as well prealloc this one's size anyway